src/main/java/core/basesyntax/Quadrangle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Quadrangle extends Figure implements Area {
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. The `draw()` and `getArea()` methods are not conceptually close to each other, so they probably shouldn't be in the same interface.